js/test/generated/beautify-javascript-tests.js

```diff
@@ -2683,6 +2683,46 @@ function run_javascript_tests(test_obj, Urlencoded, js_beautify, html_beautify,
             '?>');
         bt('a = <%= external() %>;');
 
+        //============================================================
```

Comment:
Instead of making changes here, please update the data file from which the tests are generated.